roles/contiv_network/files/v2plugin.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+set -x
```

Comment:
One other thing, could you change this to:

```sh
set -euxo pipefail
```

so it matches the rest of our scripts? 😄 Safer defaults are better defaults

Reply:
ok